src/statsig/constants.ts

```diff
@@ -28,6 +28,7 @@ export const FeatureGates = {
   [StatsigFeatureGates.SHOW_JUMPSTART_SEND]: false,
   [StatsigFeatureGates.SHOW_POINTS]: false,
   [StatsigFeatureGates.SHOW_STABLECOIN_EARN]: false,
+  [StatsigFeatureGates.COVER_STABLECOIN_EARN_GAS_FEES]: false,
```

Comment:
nit: `SUBSIDIZE` instead of `COVER`?  (it _seems_ like that's the term we've been using :shrug: )